api/v1alpha1/perses_dashboard.go

```diff
@@ -0,0 +1,20 @@
+package v1alpha1
+
+import (
```

Comment:
Not very important!
But since we're a scoped on the perses operator IMO we can avoid names like `perses_dashboard` we can call only dashboards.